db/seeds.rb

```diff
@@ -5,3 +5,5 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+User.create!(name: "Foo", email: "foo@gov.uk")
```

Comment:
I think it's better to use `find_or_create!` here so that the seed task copes with existing data and can be run multiple times.